src/components/UserActions/ClickedDocumentList.ts

```diff
@@ -146,6 +138,18 @@ export class ClickedDocumentList extends Component {
             showLessMessage: l(`${ClickedDocumentList.ID}_less`),
         });
     }
+
+    private addTooltipElement(element: HTMLElement, action: UserAction) {
+        const insertBeforeElement = element.querySelector('.CoveoResultLink');
+        if (insertBeforeElement) {
+            const tooltip = document.createElement('div');
+            tooltip.classList.add('coveo-tooltip-origin1');
+            tooltip.innerText = action.raw.origin_level_1;
```

Comment:
Are you sure it would be clear to agents that the tooltip content is the origin level 1? I think it depends a lot on whether the origin level 1 value is meaningful.

What do you think?

Reply:
I followed the mockup for that, but I think the original logic is that agents don't know more what a "search hub" or a "origin" is really, so this is more or less debug info for the educated and these people would know what this value is.

I could be mis-remembering the reasoning though.

Reply:
> ...so this is more or less debug info for the educated and these people would know what this value is

I'm not sure using a tooltip to display this information is the best choice then. My understanding is that the tooltip will be confusing for a majority of agents. To be honest, I think it will add noise for most of them.

@adroletCoveo Instead of using a tooltip, what would you think of adding an "information" icon or something that the agent could click to reveal a popup with the details? It would give some more space to indicate what this value is. More troubleshooting values could be added in the future.

Another possibility could be to have some kind of switch to enable this kind of troubleshooting information, so agents are not confused with it, and others can still access it quickly.

Any thoughts?

Reply:
No problem with these suggestions, but I would advocate they are definitely out of scope for this feature that was already planned and designed.

Perhaps in the Quantic version?

Reply:
Very good point @lbergeron  I agree that the tooltips adds noise. You have good suggestions, my only concern with the information icon is that repeating it might also add noise. The switch idea could be nice, since it's useful to only a few. I'll investigate it for the Quantic version.  
 
For this version I'll check with Antoine since he's the one who worked on the Lightning improvements, maybe the trade-off noise/usefulness is not worth it.

Reply:
Just to mention for documenting purposes, after speaking with Étienne this morning, Antoine and I think it would be better not to show the tooltip. We believe so because so far we haven't found the reason why it was needed in the first place. However it is not a terrible experience for the user and could be left as is. (but we prefer not to ;) ) 

We agreed with Étienne to follow what PMs prefer depending on the need vs time required. 
But I will think about this issue for the Quantic version of the Insight panel.